src/app-layout/index.tsx

```diff
@@ -57,6 +57,10 @@ const AppLayout = React.forwardRef(
         stickyNotifications: rest.stickyNotifications,
         disableContentHeaderOverlap: rest.disableContentHeaderOverlap,
       },
+      metadata: {
+        drawersCount: rest.drawers?.length ?? null,
```

Comment:
Any reason why it needs to be `null` and not 0 or `undefined`?

Reply:
They all have different meaning: 

* `<AppLayout drawers={[]}>` this will report zero
* `<AppLayout />` this will report null
* We cannot use `undefined`, because `JSON.stringify({value: undefined})` produces `"{}"`, so we do not know if there are no drawers or an old version without metric is used